### PR TITLE
adding instructions for Slack Alerting for Quota Monitoring Tool

### PR DESCRIPTION
**Pull request by dshah22**
Adding documentation to configure Slack Alerting for Quota Monitoring Tool and cleaning up documentation

### CODE REVIEW

File: tools/quota-monitoring-alerting/java/README.md [hunk 1]
```diff
@@ -452,7 +452,44 @@ Quota monitoring reports can be scheduled from the Data Studio dashboard using 
...
+- Custom Webhooks
+
+##### Slack Configuration
+To configure notifications to be sent to a Slack channel, you must have the Monitoring NotificationChannel Editor role on the host project.
+###### Create Notification Channel
+1. In the Cloud Console, use the project picker to select your Google Cloud project, and then select Monitoring, or click the link here: Go to Monitoring
+2. In the Monitoring navigation pane, click  Alerting.
+3. Click Edit notification channels.
+4. In the Slack section, click Add new. This brings you to the Slack sign-in page:
+&nbsp;&nbsp; a.  Select your Slack workspace.
```
Comment (boredabdel) [inline tools/quota-monitoring-alerting/java/README.md:470]: These lines will not render properly due to the non-breaking space.

Please have a look at github doc for how to indent sub-lists

https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists

File: tools/quota-monitoring-alerting/java/terraform/terraform.tfvars [hunk 2]
```diff
@@ -14,12 +14,13 @@ Copyright 20210 Google LLC
    limitations under the License.
 */
 //Update values
-project_id            = ""
-region                = ""
-service_account_email = ""
-folders               = "[]"
-organizations         = "[]"
-alert_log_bucket_name = ""
+project_id                 = ""
```
Comment (boredabdel) [inline tools/quota-monitoring-alerting/java/terraform/terraform.tfvars:17]: Can you please run a 'terraform fmt' on these lines.

File: tools/quota-monitoring-alerting/java/terraform/example.tfvars [hunk 3]
```diff
@@ -18,6 +18,7 @@ region                        = "us-west3"
 service_account_email         = "sa-quota-monitoring-project-id@quota-monitoring-project-id.iam.gserviceaccount.com"
```
Comment (boredabdel) [inline tools/quota-monitoring-alerting/java/terraform/example.tfvars:18]: Please run a 'terraform fmt' on these lines

**Review by boredabdel — COMMENTED** [review]
Few comments to address

File: tools/quota-monitoring-alerting/java/terraform/example.tfvars (hunk 3 above)
Comment (dshah22) [reply tools/quota-monitoring-alerting/java/terraform/example.tfvars:18]: done

**Review by boredabdel — APPROVED** [review]
LGTM

**Pull request merged**